### ISSUE TITLE
Make installation of the plugin easier

### ISSUE THREAD

**Issue by nbirillo** [JetBrains-Research/reflekt#70]
Currently, the user should do a lot of work to get the plugin into a Gradle build with all the configuration fiddling. Can we make it easier?

**Comment by altavir** [comment]
Yes, you can. You can use `java-gradle-plugin` plugin like it is done [here](https://github.com/mipt-npm/gradle-tools/blob/138ee1cfeaf2e80e07eec0a85288ab328f48b21e/build.gradle.kts#L4). It will make all plugin management tweaking unnecessary.

**Comment by CommanderTvis** [comment]
> Yes, you can. You can use java-gradle-plugin plugin like it is done [here](https://github.com/mipt-npm/gradle-tools/blob/138ee1cfeaf2e80e07eec0a85288ab328f48b21e/build.gradle.kts#L4). It will make all plugin management tweaking unnecessary.

This is already done by the kosogor plugin.

**Comment by nbirillo** [comment]
Thank you! It is great! We also will publish the plugin to Gradle portal after the next release. Currently it is a little bit difficult to use the space configurations
